src/io/poll/poller.rs

```diff
@@ -214,6 +214,26 @@ impl Poller {
         }
         Ok(())
     }
+    #[cfg(not(any(target_os = "macos", target_os = "freebsd", target_od = "netbsd")))]
```

Comment:
target_od = "netbsd" ➡ target_os = "netbsd"

Reply:
Oops..., thanks for your careful review!